torch_np/tests/__init__.py

```diff
@@ -0,0 +1,12 @@
+import warnings
```

Comment:
TODO: so as discussed on slack, will move move hypothesis tests to their own file and use `pytest.importorskip()`